src/Main.java

```diff
@@ -14,17 +14,15 @@ public static void main(String[] args) {
     }
 
     public static void checkLeapYear(int year) {
-
-        boolean isLeapyear = (year % 4 == 0 && year % 100 != 0 || year % 400 == 0);
-        if (isLeapyear) {
+        boolean Leapyear = year % 4 == 0 && year % 100 != 0 || year % 400 == 0;
```

Comment:
```suggestion
        boolean leapYear = year % 4 == 0 && year % 100 != 0 || year % 400 == 0;
```
camelCase соблюдать нужно